src/test.php

```diff
@@ -1,3 +1,9 @@
 <?php
 
-echo hello_world_php("Vlad");
\ No newline at end of file
+$func = FFI::cdef(
+    "int32_t numbers(int32_t arg, int32_t arg2);",
```

Comment:
When this PR will be merged we won't need FFI anymore and `numbers()` could be called as built-in PHP function

https://github.com/chayim/playground/pull/9